packages/toolkit/src/dynamicMiddleware/react/index.ts

```diff
@@ -79,7 +79,12 @@ export const createDynamicMiddleware = <
       // @ts-ignore
       context === ReactReduxContext
         ? useDefaultDispatch
-        : createDispatchHook(context)
+        : createDispatchHook(
+            context as Context<ReactReduxContextValue<
```

Comment:
i think we should just fix the parameter to actually expect a nullable context
it'll be a breaking change but technically a bug fix

Reply:
very annoying that the resolutions field meant this was missed 😕

Reply:
Agreed!